src/app/contribute/contribute-action-button.component.html

```diff
@@ -2,161 +2,161 @@
 </stache-action-buttons>
 
 <stache-page-anchor>
-  The SKY UX Team
+  The SKY UX team
 </stache-page-anchor>
 
 <sky-minifig-images
   [images]="[
     {
       name: 'Alex',
```

Comment:
Hey John. I consider these all titles, wouldn’t the current capitalization be correct?